portfolionav.html

```diff
@@ -0,0 +1,39 @@
+<html>
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="portfolio.css">
+</head>
+<body>
+	<nav >
+		 <div class="name">
+	        <ul>
+				<li>Isabelle Cochrane</li>
+			</ul>
+		</div>
+		<div class="navlinks">
+	<ul>
+	    <li>
+	        <a href=portfolionav.html>Portfolio</a>
+	    </li>
+	    <li>
+	        <a href=”/about”>About Me</a>
+	    </li>
+	    <li>
+	        <a href=”/contact”>Contact</a>
+	    </li>
+	</div>
+	  </ul>
```

Comment:
ul closing tag and div closing tag wrong way round

Reply:
fix the  <a href=”/contact”>Contact</a> part?